src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+    </div>
+
+    <div class="stars stars--1">
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
```

Comment:
Don't add empty line if siblings are single line
Just is each sibling element has several lines

```suggestion
      <div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
      
      <div class="stars stars--1">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
```